vivisect/vivbin.py

```diff
@@ -60,6 +64,22 @@ def main():
     level = e_common.LOG_LEVELS[vw.verbose]
     e_common.initLogging(logger, level=level)
 
+    if args.server_mode:
+        import vivisect.remote.server as vr_server
+        if len(args.file):
+            file = args.file[-1]
+
+        else:
+            print("Server mode requires directory name as last argument")
+            sys.exit(-1)
+
+        if args.server_port is None:
+            vr_server.runMainServer(dirname=file)
+        else:
+            server_port = int(args.server_port)
+            vr_server.runMainServer(dirname=file, port=server_port)
+        sys.exit(0)
+
```

Comment:
support VivServer within Vivbin.
i'm not opposed to making this its own script (vivserver) instead.  it works here.